snmp_cisco/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "Cisco",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "snmp_cisco",
+  "metric_prefix": "snmp.",
+  "metric_to_check": "",
```

Comment:
I think this could be one of the metrics specific to Cisco devices ? Note, this can be a list of metrics.